spec/models/event_spec.rb

```diff
@@ -10,7 +10,7 @@
 #  updated_at     :datetime
 #
 
-require 'spec_helper'
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.